apps/demo-react/util/rpc.ts

```diff
@@ -0,0 +1,6 @@
+import { CHAINS } from '@lido-sdk/constants';
```

Comment:
Please let's remove @lido-skd/constants, it's more or less deprecated, it's better to use internal variables at least

Reply:
lido-sdk will be totally removed from reef-knot soon